openshift-ansible.spec

```diff
@@ -190,6 +201,33 @@ BuildArch:     noarch
 %files lookup-plugins
 %{_datadir}/ansible_plugins/lookup_plugins
 
+# ----------------------------------------------------------------------------------
+# atomic-openshift-utils subpackage
+# ----------------------------------------------------------------------------------
+
+%package -n atomic-openshift-utils
+Summary:       Atomic OpenShift Utilities
+BuildRequires: python-setuptools
+Requires:      ansible
+Requires:      python-click
+Requires:      python-ecdsa
+Requires:      python-jinja2
+Requires:      python-markupsafe
+Requires:      python-paramiko
+Requires:      python-crypto
+Requires:      python-setuptools
+Requires:      PyYAML
+BuildArch:     noarch
+
+%description -n atomic-openshift-utils
+Atomic OpenShift Utilities includes
+ - atomic-openshift-installer
+ - other utilities
+
+%files -n atomic-openshift-utils
+%{python_sitelib}/ooinstall*
+%{_bindir}/atomic-openshift-installer
+
```

Comment:
At present the atomic-openshift-installer must be told the location of a checkout of the openshift-ansible playbooks. Biggest benefit to this merge to openshift-ansible is that this will no longer be required. I think this subpackage probably needs deps on most, if not all, of the other sub-packages to work? 

I will start to modify the actual util to expect openshift-ansible in the default RPM location if it's available, and you don't explicitly give it some other location.

This might make sense as a separate PR though after I do the above. ^^